content/docs/2.9/operate/cluster.md

```diff
@@ -15,6 +15,14 @@ However, maintainers can decide to extend this by supporting more minor versions
 > Example - At time of writing, Kubernetes 1.25 is the latest minor version so KEDA can only use new features that were introduced in 1.23
 You can learn more about the currently supported Kubernetes version in our [FAQ](https://keda.sh/docs/latest/faq/).
```

Comment:
I think we should remove this entry as well. The FAQ is not version (see https://github.com/kedacore/keda-docs/issues/936) so I think we should point from the FAQ entry to this page but can be done seperately